File: abc163/c/c.cpp
#include <iostream>
#include <vector>

using namespace std;

void manage(int n, vector<int> &a, vector<int> &res) {
  for (int i = 0; i < n - 1; i++) {
    res[a[i]] += 1;
  }
}

int main(void) {
  int n;
  cin >> n;
  vector<int> a(n + 1, 0);
  for (int i = 0; i < n - 1; i++) {
    cin >> a[i];
  }
  vector<int> res(n + 1, 0);
  manage(n, a, res);
  for (int i = 1; i < n + 1; i++) {
    cout << res[i] << "\n";
  }
  return 0;
}

File: abc163/c/c.go
package main

import "fmt"

func manage(n int, a []int) []int {
	res := make([]int, n+1)
	for _, v := range a {
		res[v] += 1
	}
	return res
}

func main() {
	var n int
	fmt.Scan(&n)
	a := make([]int, n+1)
	for i := range n - 1 {
		fmt.Scan(&a[i])
	}
	ans := manage(n, a)
	for i := 1; i <= n; i++ {
		fmt.Println(ans[i])
	}
}
